learning/tour-of-beam/frontend/lib/models/unit_content.g.dart

```diff
@@ -13,4 +13,6 @@ UnitContentModel _$UnitContentModelFromJson(Map<String, dynamic> json) =>
       description: json['description'] as String,
       taskSnippetId: json['taskSnippetId'] as String?,
       solutionSnippetId: json['solutionSnippetId'] as String?,
+      hints:
+          (json['hints'] as List<dynamic>?)?.map((e) => e as String).toList(),
```

Comment:
`json['hints']?.cast<String>()`
`(json['hints'] as List)?.cast<String>()`
?

Reply:
Is there a way to do this without editing the generated file?

Reply:
I can probably customize deserialization for `hints`, but is it worth it?

Reply:
Sorry, can't get used to generated files. :)